src/reducers/requests/posts.js

```diff
@@ -46,6 +46,17 @@ function getPostThread(state = initialRequestState(), action) {
     );
 }
 
+function getPostThreadWithRetryAttempt(state = 0, action) {
+    switch (action.type) {
+    case PostTypes.GET_POST_THREAD_WITH_RETRY_ATTEMPT:
+        return state + 1;
```

Comment:
Yep, that's right. We need to track retry attempts client side so that we know if we need to display some kind of UI element if a request is retrying.